common/test/acceptance/tests/lms/test_lms_courseware_search.py

```diff
@@ -140,30 +136,44 @@ def _studio_reindex(self):
         self.course_outline.start_reindex()
         self.course_outline.wait_for_ajax()
 
+    def _search_for_content(self, search_term):
+        """
+        Login and search for specific content
+
+        Arguments:
+
+        search_term - term to be searched for
+
+        expected - whether you expect the term to yeild results (defaults to True)
```

Comment:
I don't think expected is an argument here, seems like return value is bool result of search operation.